bugToJira.py

```diff
@@ -0,0 +1,221 @@
+#!/bin/python3
+
+import getpass
+import os
+from jira import JIRA
+import mysql.connector
+
+ANDROID_VERSION_KEY = '[ANDROID_VERSION_KEY]'
+DESCRIPTION_KEY = '[DESCRIPTION_KEY]'
+JIRA_LABEL = 'From_Bugzilla'
+
+# Functions!
+
+def get_value(question, is_mandatory=False, default_value='', is_password=False):
+    """Get a value from the user"""
+    value = ''
+    is_value_ok = False
+    while not is_value_ok:
+        if is_password:
+            value = getpass.getpass(question)
+        else:
+            value = input(question)
+        
+        if value == '' and default_value != '':
+            value = default_value
+        
+        if value == '' and is_mandatory:
+            print("This value is mandatory, again...")
+        else:
+            is_value_ok = True
+    return value
+
+# Script!
+
+print("\nWelcome to this awesome script which will migrate your bugzilla tickets to your JIRA instance. please give us all your personnal informations.\n")
+
+# Get Bugzilla DB info and connect
+mysql_host = get_value("MySQL host [localhost]: ", default_value='localhost')
+mysql_user = get_value("MySQL user [root]: ", default_value='root')
+mysql_password = get_value("MySQL password [root]: ", default_value='root', is_password=True)
+mysql_database = get_value("MySQL database table [bugzilla]: ", default_value='bugzilla')
+
+print("\nConnecting to your MySQL database...")
+mysql_conn = mysql.connector.connect(host=mysql_host, user=mysql_user,password=mysql_password, database=mysql_database)
+print("Connected.\n")
+
+# Get JIRA info and connect
+jira_username = get_value("JIRA username: ", is_mandatory=True)
+jira_password = get_value("JIRA password: ", is_mandatory=True, is_password=True)
+jira_instance = get_value("JIRA instance: https://[yourJiraInstance].atlassian.net: ", is_mandatory=True)
+jira_project = get_value("JIRA project [TEST]: ", default_value = 'TEST')
+
+print("\nConnecting to your JIRA instance...")
+jira_options = {'server': 'https://' + jira_instance + '.atlassian.net'}
+jira = JIRA(jira_options, basic_auth=(jira_username, jira_password))
+print("Connected.\n")
+
+# Get Zendesk instance to create cool links
+zendesk_instance = get_value("Zendesk instance: https://[yourZendeskInstance].zendesk.com: ", is_mandatory=True)
+
+# Load bugs from bugzilla
+print("\nLoading bugs...")
+cursor_bugs = mysql_conn.cursor()
+cursor_bugs.execute("""
+SELECT 
+b.bug_id,
+b.bug_severity,
+b.bug_status,
+b.creation_ts,
+b.short_desc,
+b.op_sys,
+b.priority,
+p.name product,
+b.rep_platform,
+b.version,
+c.name component,
+b.cf_bugtype,
+b.cf_listusers,
+b.cf_zendesk_ticket_id_text
+
+FROM bugs b, products p, components c
+
+WHERE
+b.bug_status NOT IN ('RELEASED', 'CLOSED')
```

Comment:
This status has been accepted by Support team and @Pascal. I'll triple check with @Thomas